docs/features/enhanced-using.md

```diff
@@ -0,0 +1,83 @@
+# Enhanced Using
+
+Enhanced using consists to two related features that aim to make the ```IDisposable``` coding pattern easier to participate in for implementers, and easier to consume for end users.
+
+__See__: the [corresponding proposal](https://github.com/dotnet/csharplang/blob/d2ce4cc3e17708e7e1d062bf40da0901a744fa3b/proposals/using.md) in CSharpLang.
+
+_Pattern based dispose_ aims to make participation in the disposable pattern easier by relaxing the restrictions on what types can be used in a ```using(...){ }``` construct. 
+
+_Using declarations_ aim to make consuming a disposable type simpler by allowing ```using``` to be added to a variable declaration. 
+
+
+## Pattern based dispose
+
+Previously a type must derive from ```IDiposable``` / ```IAsyncDisposable``` and implement the ```void Dispose()``` / ```Task DisposeAsync()``` methods. With pattern-based dispose, a type can be considered 'disposable' if it meets certain structural requirements. Specifically:
```

Comment:
You're missing something at the end of the first sentence. The sentence starts out with "Previously ..." but doesn't have a conclusion on what could be done previously.